neuralmonkey/runners/label_runner.py

```diff
@@ -20,7 +20,10 @@ def __init__(self,
         # make sure the lazy decoder creates its output tensor
         log("Decoder output tensor: {}".format(decoder.decoded))
 
-    def get_executable(self, compute_losses=False, summaries=True):
+    def get_executable(self,
+                       compute_losses=False,
+                       summaries=True,
+                       num_sessions=1):
```

Comment:
Proč potřebujeme vědět počet sessions takovýhle způsobem?

Reply:
Pokud to mají mít všechny runnery a executably, přesunul bych to někam do předka.